pylxca/test/pylxca_unittest

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env /usr/bin/python2.7
+#!/usr/bin/env /usr/bin/python
```

Comment:
I think you have to do 

```suggestion
#!/usr/bin/env python3
```

Reply:
Ah yeah I doubled up the `/usr/bin` my bad will fix.  I think `/usr/bin/env python` is still preferred because it allows for you to use both, and build.py is still `/usr/bin/env python` (https://github.com/lenovo/pylxca/blob/master/build.py#L1)

I think to set this to `/usr/bin/env python3` we'd have to change that as well.